### ISSUE TITLE
JointGenotypingTasks.wdl File to String coercion produce an error

### ISSUE THREAD

**Issue by EugeneEA** [broadinstitute/warp#783]
task `GenotypeGVCFs` in file  JointGenotypingTasks.wdl has an input `String dbsnp_vcf` but it should be a `File dbsnp_vcf` as correctly declared in inputs of workflow JointGenotyping.wdl. Coercion of file to string produces an obviouse error as far as correct file does not hardlinked to  the mounted docker container

Best, Eugene

**Comment by kevinpalis** [comment]
Hi @EugeneEA, is this still causing issues for you? If it is, can you paste a copy of the error you're seeing and provide more details on how you're running the pipeline? We often specify files as strings in tasks to avoid unnecessary localization.

**Comment by kevinpalis** [comment]
Hi @EugeneEA, from the paths that you provided, it looks like you're running this locally or on a baremetal server? Are you running cromwell locally or on your own server? It should try to localize strings when you need them but `file:///home-local/source_files/hg38/hg38.dbsnp.vcf.gz` looks a bit malformed (extra '/' in the beginning). Can you try removing the first '/' in `/home-local/source_files/hg38/hg38.dbsnp.vcf.gz` and try running it again (as String, not File)?

**Comment by jessicaway** [comment]
@EugeneEA did youi have any success with the suggestion above?

**Comment by EugeneEA** [comment]
> @EugeneEA did you have any success with the suggestion above?

Hi, sorry for not responding. Indeed as I wrote in my original post I fixed my problem by changing String to File.
Yes I'm running wdl on a local server and it seems that Cromwell makes a mistake while "localizing" the string (probably by adding additional '/' while parsing the input.json). But I have not check it as far as solved the problem differently. I just wanted to point to the problem. Issue might be closed, Thanks for the feedback!
